pkgs/development/python-modules/jq/jq-py-setup.patch

```diff
@@ -0,0 +1,130 @@
+From 3f369cf8b9f7134d0792f6b141d39b5342a8274f Mon Sep 17 00:00:00 2001
+From: Benjamin Staffin <benley@gmail.com>
+Date: Mon, 14 Jan 2019 17:27:06 -0500
+Subject: [PATCH] Vastly simplify setup.py for distro compatibility
```

Comment:
Is this patch only needed for Nix?

Reply:
Nvm, read the OP again
> Python bindings for jq. This requires patching setup.py to avoid it doing nasty things,

So not intended for submission.

Reply:
yeah - a similar patch will be needed for pretty much any distribution, but it's unlikely to be accepted upstream.